.readthedocs.yaml

```diff
@@ -3,11 +3,11 @@ version: 2
 
 
 build:
-  os: ubuntu-22.04
+  os: ubuntu-lts-latest
```

Comment:
Why do we need this change? GitHub CI is still running on Ubuntu 20.04 without issues.

Reply:
I may be misunderstanding, but this version is run by Read the Docs?

Reply:
This, RTD does a separate run, but only building the docs, not running the tests for us.

Reply:
If they run the docs on the latest version of Ubuntu and it fails, this may be useful to know.

Reply:
If we would have OS-specific code in our own code base, I would understand this. But we only have this in our tests which are not being run on RTD anyway, does I do not consider this essential.

Reply:
Reverted the change.

I am trying to fix Read the Docs is outdated #2484.